kernel/aster-nix/src/fs/utils/page_cache.rs

```diff
@@ -319,31 +319,30 @@ impl PageCacheManager {
     pub fn evict_range(&self, range: Range<usize>) -> Result<()> {
         let page_idx_range = get_page_idx_range(&range);
 
-        //TODO: When there are many pages, we should submit them in batches of folios rather than all at once.
```

Comment:
Good point. Although we have the ability to have multiple `BioSegment`s in a `Bio`, which can make folio work. But the API now in `BlockDevice` only take one `BioSegment` to submit a bio request once.
So I suggest we can add a new API like `write_blocks(&self, bid: Bid, segments: &[Segment]) -> Result<BioWaiter, BioEnqueueError>()` to accomplish this. If so, this will be done in the next PR.